spec/tests/result.fmf

```diff
@@ -20,30 +20,47 @@ description: |
         ignore the actual test result and always report provided
         value instead
     custom
-        test needs to create it's own ``results.yaml`` or
+        test needs to create its own ``results.yaml`` or
         ``results.json`` file in the ``${TMT_TEST_DATA}``
-        directory. The format of the file is documented at
-        :ref:`/spec/plans/results`. When importing results from
-        the custom file, each test name, referenced in the file
-        via ``name`` key, would be prefixed by the original test
-        name. A special case, ``name: /``, sets the result for the
+        directory. The format of the file, notes and detailed examples
+        are documented at :ref:`/spec/plans/results`.
+
+      .. Duplication warning: do NOT edit the following paragraph, always edit
+         the correspondign one in /spec/plans/results.fmf, then copy & paste it
+         here.
+
+         The goal is to treat results.fmf as the main source of truth, and while
+         it's useful to show this paragraph here as well, let's strive to keep
+         it in sync.
+
+        When importing the custom results file, each test name referenced
+        in the file by the ``name`` key would be prefixed by the original
+        test name. A special case, ``name: /``, sets the result for the
         original test itself.
 
 example:
   - |
     # Plain swapping fail to pass and pass to fail result
     result: xfail
+
+  # Duplication warning: do NOT edit the following examples, always edit
+  # examples in /spec/plans/results.fmf, then copy & paste those here.
```

Comment:
Regarding the duplication I'd agree with @lukaszachy: Let's keep all the details related to the `results.yaml` and `results.json` format on a single page and remove it from here. Or, perhaps, just keep the super short stingy example to get a quick idea what it is about.

Reply:
Hopefully addressed in 9cea19ca - duplication is gone, and a trivial `custom` example was added with a comment.

Reply:
Super, now looks great! Thanks.